Mutation pages updates (#185)
neo4j/docs-graphql: modules/ROOT/pages/mutations/create.adoc

}
----

> This creates a `User` with name "John Doe".

Reviewer: ```suggestion
This creates a `User` with the name "John Doe".
```

Not sure if this is technically needed, but it seems more correct to me.

The name and the autogenerated ID are returned.

== Nested `create`

> You can create a `User` and his initial `Post` at once by executing the following:

Reviewer: ```suggestion
You can create a `User` and their initial `Post` at once by executing the following:
```

== `connectOrCreate` relationships

> If a related node has the `@unique` directive defined, you can use `connectOrCreate` nested in a `create` to perform an operation similar to a xref:https://neo4j.com/docs/cypher-manual/current/clauses/merge/[Cypher `MERGE`] operation on the related node.

Reviewer 2: Maybe worth specifying what this `create` is? I don't know if it's a mutation, but just to give an example and also fix the xref to link

```suggestion
If a related node has the `@unique` directive defined, you can use `connectOrCreate` nested in a `CREATE` mutation to perform an operation similar to a link:https://neo4j.com/docs/cypher-manual/current/clauses/merge/[Cypher `MERGE`] operation on the related node.
```
